test/categories/test_shape.py

```diff
@@ -0,0 +1,15 @@
+import unittest
+from qual_id.categories.shape import Shape
+import random
+
+
+class TestShape(unittest.TestCase):
+  def setUp(self):
+    self.shape = Shape()
+
+  def test__get_values__is_valid(self):
+        self.assertTrue(CategoryHelper.get_values_is_valid(self.shape))
```

Comment:
As you can see from the error message in the tests, you need to import `CategoryHelper`, just like you did [here](https://github.com/gabrielbarker/qual-id/blob/main/test/categories/test_color.py)
Also you don't need `import random`

Reply:
sorry, i will fix it